src/steam-user/steam-user.service.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  BadRequestException,
+  Injectable,
+  NotFoundException,
+} from '@nestjs/common';
+import { CreateSteamUserDto } from './dto/create-steam-user.dto';
+import { UpdateSteamUserDto } from './dto/update-steam-user.dto';
+import { PrismaService } from 'src/prisma/prisma.service';
+
+@Injectable()
+export class SteamUserService {
+  constructor(private readonly prisma: PrismaService) {}
+
+  async create(createSteamUserDto: CreateSteamUserDto) {
+    const oldUser = await this.prisma.steamUser.findUnique({
+      where: { steamid: createSteamUserDto.steamid },
+    });
```

Comment:
This value is never used. I guess you wanted to throw an error if `oldUser` exists